src/cli/build.rs

```diff
@@ -112,7 +152,6 @@ impl PreBuild {
             .lines()
             .filter_map(|line| line.ok())
             .for_each(|line| println!("{}", line));
-        return;
```

Comment:
These changes should not be part of that commit, but you should apply them to the commits where you add those return.